src/test/ua/testbiz/page/CandidateAutomationTest.java

```diff
@@ -0,0 +1,29 @@
+package ua.testbiz.page;
+
+import data.Candidate;
+import org.testng.Assert;
+
+public class CandidateAutomationTest extends BaseTest {
+
+    @org.testng.annotations.Test
+    public void addCandidate() {
+        LoginPage loginPage = new LoginPage(driver);
+        HomePage homePage = new HomePage(driver);
+        RecruitmentPage recruitmentPage = new RecruitmentPage(driver);
+
+        loginPage.loginIntoSystem();
+        Assert.assertTrue(homePage.isOpened(), "Not a home page");
+
+        homePage.selectRecruitment();
+
+        recruitmentPage.switchToCandidate();
+
+        Candidate candidate = new Candidate();
+        candidate.setCandidateInfo();
+        recruitmentPage.addCandidate(candidate);
+
+        recruitmentPage.switchToCandidate();
+
+        Assert.assertNotNull(recruitmentPage.checkCandidate(candidate), "Candidate is not found");
+    }
```

Comment:
Assert cannot be between steps inside of method. You can split it on separate methods and each of them should contain Assert as last row.

Reply:
Fixed